charts/clamav/templates/clamav-mirror-configmap.yaml

```diff
@@ -23,15 +23,27 @@ data:
     )
     index-file.names = ( "index.htm" )
     server.modules += ( "mod_accesslog" )
-    accesslog.filename  = "/dev/stderr" 
+    accesslog.filename  = "/dev/stderr"
+    $HTTP["url"] == "/server-status" { 
+      accesslog.filename = "/dev/null"
+    }
+    $HTTP["useragent"] =~ "kube-probe" {
+      accesslog.filename = "/dev/null"
+    }
```

Comment:
This filters out logs by Sysdig "/server-status" and kube-probe, so mirror logs are cleaner

Reply:
Should be a comment?

Reply:
Added comments to make it clearer, tell me if I misunderstood

Reply:
👍